.travis.yml

```diff
@@ -20,6 +20,13 @@ script:
     fi
 
 before_deploy:
+  - |
```

Comment:
I'm not 100% sure if the `~/.aws` dir will exist at this point so it may be worth adding in a step to ensure it exists `mkdir -p ~/.aws`

Reply:
done